### PR TITLE
Fix #2343

### LINKED ISSUE #2343: ClearrMLLogger freezes logging in distributed env
## 🐛 Bug description

Instantiating a `ClearMLLogger`instance in a distributed training freezes the logging of the subprocesses. The bug can be reproduced by executing the following script: https://github.com/H4dr1en/trains/commit/642c1130ad1f76db10ed9b8e1a4ff0fd7e45b3cc

I will open a PR to fix it.

## Environment

- PyTorch Version (e.g., 1.7.1):
- Ignite Version (e.g., 0.5.0):
- OS (e.g., Linux): linux
- How you installed Ignite (`conda`, `pip`, source): pip/source
- Python version: 3.8.5
- Any other relevant information:

### PR DESCRIPTION
**Pull request by H4dr1en**
Fixes #2343 

Description: ClearMLLogger tries to retrieve current task before trying to create a new one.

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [X] Documentation is updated (if required)

### CODE REVIEW

**Comment by vfdev-5** [comment]
@H4dr1en thanks for the PR ! Do you think we can write a distributed test for checking that everything is working ?

File: ignite/contrib/handlers/clearml_logger.py [hunk 1]
```diff
@@ -147,12 +148,14 @@ def __setattr__(self, attr: str, val: Any) -> None:
 
             self._task = _Stub()
         else:
-            self._task = Task.init(
-                project_name=kwargs.get("project_name"),
-                task_name=kwargs.get("task_name"),
-                task_type=kwargs.get("task_type", Task.TaskTypes.training),
-                **experiment_kwargs,
-            )
+            self._task = Task.current_task()
```
Comment (vfdev-5) [inline ignite/contrib/handlers/clearml_logger.py:151]: Could you please leave a comment here on what we are trying to do like the PR's description:
"ClearMLLogger tries to retrieve current task before trying to create a new one." ?

**Comment by H4dr1en** [comment]
@vfdev-5 This might be very tricky to test: we want to make sure that the communication between the subprocesses and the main process works well, so that clearml can log successfully in the clearml-server. I have now idea how to test that :D

**Review by vfdev-5 — APPROVED** [review]
@H4dr1en thanks a lot for the PR ! 
For the tests I agree that this could be a bit tricky to write and there is no error to catch. 
Let's merge this like that and if possible to add a test we can do that later in a follow-up PR.
I still would like to hear from ClearML team.
@bmartinn could you please review as well. Thanks !


EDIT: There is a lint to fix and TPU tests are unrelated

**Pull request merged**